.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
```

Comment:
It is totally arbitrary what a user will call their build folder. I don't expect we can add a specific user environment directories into our repo.

Reply:
If the user wants git to ignore the build directory its best not to place the build directory in the source directory.